airflow/executors/local_executor.py

```diff
@@ -383,6 +383,10 @@ def end(self) -> None:
             raise AirflowException(NOT_STARTED_MESSAGE)
         if not self.manager:
             raise AirflowException(NOT_STARTED_MESSAGE)
+        self.log.info(
+            "Shutting down LocalExecutor"
+            " - this will wait for running tasks to finish, or signal again if you don't want to wait."
+        )
```

Comment:
```suggestion
        self.log.info(
            "Attempting to shut down LocalExecutor"
            "; waiting for running tasks to finish.  Signal again if you don't want to wait."
        )
```

main point here is to clarify grammatically that it's not `this will (wait or signal)` but `(this will wait) and (you must signal if...)`

Reply:
Good call, thanks. I left beginning alone though.